screens/MainLayout.tsx

```diff
@@ -77,6 +81,13 @@ export const MainLayout: React.FC<RootRouteProps> = (props) => {
           key={route.name}
           name={route.name}
           component={route.component}
+          listeners={{
+            tabPress: (e) => {
+              if (route.name == 'Scan') {
```

Comment:
use constant from routes config instead of string

Reply:
Done changed it to scan.name